documentation/installation/manual_cli.md

```diff
@@ -0,0 +1,37 @@
+---
+id: manual_cli
+title: Installation, Linux command-line
+description: "Manual installation steps using the Linux command-line"
+---
+
+This guide will help you install HACS using the Linux command-line on your Home Assistant server.
+
+### Step 1 - Go to your Home Assistant configuration directory 
+
+The `hacs` folder needs to be placed under `<config_dir>/custom_components/`
+
+`<config-dir>` is where your `configuration.yaml` file is located.
+
+On Hassio/Docker the final location will be `/config/custom_components/hacs`
+
+On Hassbian(venv) the final location will be `/home/homeassistant/.homeassistant/custom_components/hacs`
+
+For this example, `<config-dir>` will be `/path/to/homeassistant/config`
+
+Change-directory to your Home Assistant configuration directory:
+```
+cd /path/to/homeassistant/config
```

Comment:
This part is not needed, the script handles it.

If no dir is found, the script will tell the user to do this as well.